index.html

```diff
@@ -0,0 +1,880 @@
+              alt="Cat"
+            />
+          </div>
+
+          <a href="#" class="adds-title">1,651 smashing members</a>
+          <p>
+            A friendly community that keeps this website alive. With books,
+            webinars, discounts and networking — for just 1 coffee a month.
+          </p>
+          <a href="#">Join the family.</a
+          ><span> <a href="#" class="adds-btn">Explore features →</a></span>
+          <small>Already have an account? <a href="#">Sign in!</a></small>
+        </div>
+
+        <div class="post-text">
+          <h4>2 DAYS AGO</h4>
+          <h3>
+            Smashing Postcast Eposode 11 With Eduardo Boucas: What is Sourcebit?
+          </h3>
+          <span> by <a href="#">Drew McLellan</a></span>
+          <p>
+            In this episode of the Smashing Podcast we’re talking about an
+            interesting open-source tool called Sourcebit. How can it help our
+            content workflow with JAMstack sites? Drew McLellan speaks to
+            developer Eduardo Bouças to find out.
+          </p>
+          <i>→</i>
+        </div>
+        <div class="post-text">
+          <h4>2 DAYS AGO</h4>
+          <h3>
+            Smashing Postcast Eposode 11 With Eduardo Boucas: What is Sourcebit?
+          </h3>
+          <span> by <a href="#">Drew McLellan</a></span>
+          <p>
+            In this episode of the Smashing Podcast we’re talking about an
+            interesting open-source tool called Sourcebit. How can it help our
+            content workflow with JAMstack sites? Drew McLellan speaks to
+            developer Eduardo Bouças to find out.
+          </p>
+          <i>→</i>
+        </div>
+        <div class="post-text">
+          <h4>2 DAYS AGO</h4>
+          <h3>
+            Smashing Postcast Eposode 11 With Eduardo Boucas: What is Sourcebit?
+          </h3>
+          <span> by <a href="#">Drew McLellan</a></span>
+          <p>
+            In this episode of the Smashing Podcast we’re talking about an
+            interesting open-source tool called Sourcebit. How can it help our
+            content workflow with JAMstack sites? Drew McLellan speaks to
+            developer Eduardo Bouças to find out.
+          </p>
+          <i>→</i>
+        </div>
+        <div class="post-text">
+          <h4>2 DAYS AGO</h4>
+          <h3>
+            Smashing Postcast Eposode 11 With Eduardo Boucas: What is Sourcebit?
+          </h3>
+          <span> by <a href="#">Drew McLellan</a></span>
+          <p>
+            In this episode of the Smashing Podcast we’re talking about an
+            interesting open-source tool called Sourcebit. How can it help our
+            content workflow with JAMstack sites? Drew McLellan speaks to
+            developer Eduardo Bouças to find out.
+          </p>
+          <i>→</i>
+        </div>
```

Comment:
You can keep the text in this part, since it's needed, but use the same grayscale criteria as stated above.

The images should be divs with a grayscale background depending on their importance